### ISSUE TITLE
HTML report does not contain source listing in 1.4.8

### ISSUE THREAD

**Issue by davidburstrom** [szpak/gradle-pitest-plugin#198]
There's been a regression between 1.4.7 and 1.4.8 in the HTML reporting, when looking at individual classes. In 1.4.7 the report still contained the source code of the mutated production classes, but in 1.4.8 that went away. The report still lists the performed mutations and line numbers.

**Comment by szpak** [comment]
Thanks for your report. Do you use the same PIT version (overriden by `pitestVersion`) or the default ones?

**Comment by davidburstrom** [comment]
I've tried a permutation of PITest [1.5.0, 1.5.1] and plugin [1.4.7, 1.4.8], and the source code is omitted as soon as 1.4.8 is used.

**Comment by szpak** [comment]
Thanks. It's probably [this commit](https://github.com/szpak/gradle-pitest-plugin/commit/f4159ede317a50aa13336c7ab3b18d327c7b76b5). I will write functional test to verify that and I will try to fix it.

**Comment by szpak** [comment]
@davidburstrom You may want to play with 1.4.9-SNAPSHOT to check if it fixed the problem for you.

```
buildscript {
    repositories {
        mavenCentral()
        //Needed only for SNAPSHOT versions
        maven { url 'https://oss.sonatype.org/content/repositories/snapshots/' }
    }
    dependencies {
        classpath 'info.solidsoft.gradle.pitest:gradle-pitest-plugin:1.4.9-SNAPSHOT'
    }
}
```

**Comment by davidburstrom** [comment]
I'm inclined to say that it is indeed fixed in 1.4.9-SNAPSHOT. Well done!

**Comment by szpak** [comment]
Thanks. 1.4.9 just has been released.
